pandas/core/groupby/groupby.py

```diff
@@ -529,6 +529,9 @@ class providing the base-class of operations.
     - string function name
     - list of functions and/or function names, e.g. ``[np.sum, 'mean']``
     - dict of axis labels -> functions, function names or list of such.
+    - None, in which case output is determined from ``**kwargs`` using Named Aggregation. In this scenario the output
+      has one column for each element in ``**kwargs``. The name of the column is a string of the keyword's variable name. The
```

Comment:
I think from the Ci checks that the line lengths here are too long: https://github.com/pandas-dev/pandas/actions/runs/3102809189/jobs/5025470354